src/js/dataMapping/covid19/awardSpendingAgency/awardSpendingAgencyTableTabs.js

```diff
@@ -0,0 +1,67 @@
+export const awardSpendingAgencyTableTabs = [
+    {
+        enabled: true,
+        internal: 'all',
+        label: 'All Awards'
+    },
+    {
+        enabled: true,
+        internal: 'contracts',
+        label: 'Contracts'
+    },
+    {
+        enabled: true,
+        internal: 'idvs',
+        label: 'Contract IDVs'
+    },
+    {
+        enabled: true,
+        internal: 'grants',
+        label: 'Grants'
+    },
+    {
+        enabled: true,
+        internal: 'direct_payments',
+        label: 'Direct Payments'
+    },
+    {
+        enabled: true,
+        internal: 'loans',
+        label: 'Loans'
+    },
+    {
+        enabled: true,
```

Comment:
Again, I would remove all of these tabs that aren't in the final design

Reply:
Done!